src/libxdata/XData.cpp

```diff
@@ -66,7 +67,21 @@ void XData::cancelLoading() {
 }
 
 void XData::loadAirports() {
-    AirportLoader loader(world);
+    const AirportLoader loader(world);
+
+    //Custom scenery
+    CustomSceneryParser parser(xplaneRoot + "Custom Scenery/scenery_packs.ini");
+    parser.setAcceptor([this,loader] (const std::string &data) {
+        auto customScenaryDir = xplaneRoot + data + "/Earth nav data/apt.dat";
+        if(!platform::fileExists(customScenaryDir))
+            return;
+
+        logger::info("Loading custom scenary airport for %s", data.c_str());

```

Comment:
"scenery" misspelled.

Reply:
Well thats embarrassing, for some reason my brain struggles with the word "scen(e|a)ry" 😄